### PR TITLE
Bump serverless images

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -82,13 +82,13 @@ images:
   runtimes:
     nodejs12:
       repository: "eu.gcr.io/kyma-project/function-runtime-nodejs12"
-      tag: "5daaf5c9"
```
Comment (kwiatekus) [inline resources/serverless/values.yaml:85]: I'm no expert but shouldn't those be bumped? 
eu.gcr.io/kyma-project/function-webhook:f4edbb0a
eu.gcr.io/kyma-project/function-build-init:f4edbb0a
eu.gcr.io/kyma-project/function-controller:f4edbb0a

Comment (m00g3n) [reply resources/serverless/values.yaml:85]: sorry my bad

**Comment by kwiatekus** [comment]
When running kyma from this PR console UI doesnt render function details

**Pull request closed without merging**